scripts/src/server.ts

```diff
@@ -19,6 +19,8 @@ import { ExtendedArray } from "./common/array";
 
 import { middleware as statics } from "./middleware/statics";
 import { middleware as logging } from "./middleware/logging";
+import {Config as SessionConfig, middleware as session, Session} from "./middleware/session";
```

Comment:
(formatting)
pad it with spaces:
`{ Config as SessionConfig, middleware as session, Session }`